lib/Player.tsx

```diff
@@ -345,6 +333,15 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
                 toggleStats={toggleStatsOverlay}
               />
             </Layer>
+            {showStatsOverlay && videoProperties !== undefined ? (
+              <Stats
+                api={api}
```

Comment:
Should not just be a string (I see it was already typed wrong before).

Reply:
`api` is string in several places. I guess it would be better to change them all at once in a different PR.